components/learn/TheLearnHeader.vue

```diff
@@ -1,16 +1,20 @@
 <template>
   <ThePageHeader class="the-learn-header">
-    Start your path towards<br>
-    learning <RelevantWord>Qiskit</RelevantWord>
+    Start your path towards
+    <br />learning
+    <TypedWord
+      :initialWord="'Qiskit'"
```

Comment:
Use kebab-case here, it is automatically converted into camelCase by Nuxt.js.

Reply:
Also, you don't need this to be dynamic so it is better to use:
```suggestion
      :initial-word="Qiskit"
```